src/Illuminate/Database/Eloquent/Relations/HasOneOrMany.php

```diff
@@ -217,14 +217,18 @@ public function save(Model $model)
     /**
      * Attach an array of models to the parent instance.
      *
-     * @param  array  $models
+     * @param  array|Collection $models
```

Comment:
Should be either `mixed` either `array|\Illuminate\Database\Eloquent\Collection`
This is not a `\Illuminate\Support\Collection`